test/base_test_fixture.h

```diff
@@ -1502,6 +1502,38 @@ struct base_test_fixture
         results.get_ref(0, ref);
         REQUIRE(ref == name);
     }
+    void string_vector_test()
```

Comment:
Yes, there is a bit of fiddling.

Basically,
* The `base_test_fixture.h` is a set of common test cases.
* Each `<database>_test.cpp` is a source code for an independent test program/runner that includes common test cases, plus database-specific cases if such cases exist/are necessary to add.

The procedure is this:
1. Add common test case to `base_test_fixture.h`
2. Copy & paste the `TEST_CASE_METHOD` bolierplate in each of `<database>_test.cpp` files separately, updating name/tags, etc.

If, a feature requires very specific test cases for each database, then skip `base_test_fixture.h` and write dedicated test case in each `<database>_test.cpp` file.

I hope it helps.